sleap_nn/training/utils.py

```
@@ -5,9 +5,173 @@
+    UNetLargeRFConfig,
+    ConvNextSmallConfig,
+    ConvNextLargeConfig,
+    ConvNextBaseConfig,
+    SwinTBaseConfig,
+    SwinTSmallConfig,
+    SingleInstanceConfig,
+    SingleInstanceConfMapsConfig,
+    CentroidConfig,
+    CenteredInstanceConfMapsConfig,
+    CenteredInstanceConfig,
+    CenteredInstanceConfMapsConfig,
```

Ruff (0.8.2)

Redefinition of unused `CenteredInstanceConfMapsConfig` from line 29

Remove definition: `CenteredInstanceConfMapsConfig`

(F811)